artifactory/services/upload.go

```diff
@@ -26,7 +26,6 @@ type UploadService struct {
 	DryRun            bool
 	Threads           int
 	MinChecksumDeploy int64
-	Retries           int
```

Comment:
The *Retries* option has been removed from the upload service. 
This causes a bit of inconsistency with the download service, which still includes the *Retries* option.
I think that it make more sense for the *Retries* to be set in the service level than in the upload / download method level, but we can discuss this further.

Reply:
This will be fixed in a future development which redesigns the way of working with services and params.